cloud_utils/scheduler/kubernetes_connector.py

```diff
@@ -85,14 +85,22 @@ def create_cron_job(
         ),
     )
     try:
-        options = {"namespace": "default", "body": cron_job, "pretty": "true"}
+        options = {"name": f"{pod_name}-cronjob", "namespace": "default", "body": cron_job, "pretty": "true"}
```

Comment:
I think this can be written with less duplication.

create a function that creates the options and in the first case just use toolz.assoc to add the name.